### DISCUSSION TITLE
Works on MacOS

### DISCUSSION THREAD

**Discussion by peter-dye** [kuiperzone/AvantGarde#16] [Announcements]
This isn't an actual issue but I wanted to let you know that I cloned the code and built on MacOS (Intel CPU) and so far, it seems to be working! I haven't tried anything too advanced, but it previews a newly created Avalonia project fine.

This is pretty awesome. I didn't want to buy Rider to have axaml previews while developing on my Mac and this looks like it will solve that problem!

**Comment by kuiperzone** [comment]
Great stuff!

I think it's as much a testament to Avalonia itself that apps work cross-platform out of the box.

**Reply by MAXMASTER001** [reply]
It worked on my M1 macbook too so far.

**Comment by kuiperzone** [comment]
Guys, did you have to do any regarding adding `dotnet` to the `$PATH` in order to get it to work on Mac?

**Reply by peter-dye** [reply]
I do have dotnet in my path, now sure if it was required there to work.
